cit-api/pipeline/migrations/0147_auto_20210413_1615.py

```diff
@@ -0,0 +1,413 @@
+# Generated by Django 2.2.16 on 2021-04-13 16:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pipeline', '0146_firstnationshealthauthoritysite'),
+    ]
+
+    operations = [
+        migrations.RunSQL("""CREATE OR REPLACE VIEW public.cit_opportunities_vw
+        AS
+        SELECT o.id AS opportunity_id,
+            o.opportunity_address,
+            st_y(o.geo_position) AS latitude,
+            st_x(o.geo_position) AS longitude,
+            o.date_created,
+            o.date_updated,
+            ( SELECT a.status_name AS approval_status_name
+                FROM pipeline_approvalstatus a
+                WHERE a.status_code::text = o.approval_status_id::text) AS approval_status_name,
+            ( SELECT a.status_description AS approval_status_description
+                FROM pipeline_approvalstatus a
+                WHERE a.status_code::text = o.approval_status_id::text) AS approval_status_description,
+            ( SELECT a.active_status AS approval_status_active_ind
+                FROM pipeline_approvalstatus a
+                WHERE a.status_code::text = o.approval_status_id::text) AS approval_status_active_ind,
+            o.business_contact_email,
+            o.business_contact_name,
+            o.community_link AS community_url,
+            o.elevation_at_location AS lation_elevation,
```

Comment:
Is `lation_elevation` a typo, or is that intentional?

Reply:
This is on the BCGW side. For now lets keep as expected